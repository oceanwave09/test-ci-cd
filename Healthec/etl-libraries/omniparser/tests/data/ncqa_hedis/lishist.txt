D        95002                                    2021010120211231                                                                                                   
D        95003                                    20210101                                                                                                           
D        95004                                    2019062520210909                                                                                                   
D        95005                                    2019040920200515                                                                                                   
D        95006                                    2023122820220217                                                                                                   