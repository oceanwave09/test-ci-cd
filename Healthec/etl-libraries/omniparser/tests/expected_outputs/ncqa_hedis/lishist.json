[
	{
		"beneficiary_id": "95002",
		"low_income_period_end_date": "20211231",
		"low_income_period_start_date": "20210101",
		"record_type": "D",
		"row_id": "e53af966-8c9d-4365-b44a-3359d44c3db8"
	},
	{
		"beneficiary_id": "95003",
		"low_income_period_start_date": "20210101",
		"record_type": "D",
		"row_id": "6822a85f-ba98-4468-84c5-e4e1f13a34eb"
	},
	{
		"beneficiary_id": "95004",
		"low_income_period_end_date": "20210909",
		"low_income_period_start_date": "20190625",
		"record_type": "D",
		"row_id": "41b97942-8fff-45bb-9a8c-62fe5ba1a953"
	},
	{
		"beneficiary_id": "95005",
		"low_income_period_end_date": "20200515",
		"low_income_period_start_date": "20190409",
		"record_type": "D",
		"row_id": "a9cdb3ba-10ab-4d0b-a796-000e98de790d"
	},
	{
		"beneficiary_id": "95006",
		"low_income_period_end_date": "20220217",
		"low_income_period_start_date": "20231228",
		"record_type": "D",
		"row_id": "92c52c04-f7d4-4c25-93a7-1069e99687fc"
	}
]
